inbc-program/tests/unit/test_source_app_parser.py

```diff
@@ -24,8 +26,76 @@ def test_parse_add_arguments_successfully(self):
                                      'deb-src http://example.com/ focal-security main'])
         self.assertEqual(f.filename, 'intel-gpu-jammy.list')
 
+    def test_parse_add_arguments_deb822_format_separate_lines_successfully(self):
+        f = self.arg_parser.parse_args(
+            ['source', 'application', 'add',
+             '--sources', 'X-Repolib-Name: Google Chrome',
+                          'Enabled: yes',
+                          'Types: deb',
+                          'URIs: https://dl-ssl.google.com/linux/linux_signing_key.pub',
+                          'Suites: stable',
+                          'Components: main',
+             '--filename', 'intel-gpu-jammy.list'])
```

Comment:
**Suggestion:** Consider using a loop to iterate over the 'sources' list to avoid code repetition and improve maintainability. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        sources_list = [
            'X-Repolib-Name: Google Chrome',
            'Enabled: yes',
            'Types: deb',
            'URIs: https://dl-ssl.google.com/linux/linux_signing_key.pub',
            'Suites: stable',
            'Components: main'
        ]
        f = self.arg_parser.parse_args(
            ['source', 'application', 'add',
             '--sources'] + sources_list +
            ['--filename', 'intel-gpu-jammy.list'])
```